packages/app/src/runner/selector-playground/SelectorPlayground.vue

```diff
@@ -1,83 +1,135 @@
 <template>
   <div
     id="selector-playground"
-    class="bg-white flex items-center"
+    class="border-t border-b bg-gray-50 border-gray-200 h-56px grid p-12px gap-12px grid-cols-[40px,1fr,auto] items-center "
   >
     <button
-      :class="{ 'bg-blue-100': selectorPlaygroundStore.isEnabled }"
-      class="rounded-md h-full px-8px"
+      class="border rounded-md flex h-full outline-none text-white transition w-40px duration-150 items-center justify-center"
+      :class="{'default-ring': selectorPlaygroundStore.isEnabled, 'border-gray-200': !selectorPlaygroundStore.isEnabled}"
```

Comment:
If you are interested there's a ternary syntax, too:

```html
<button
  :class="[ selectorPlaygroundStore.isEnabled ? 'default-ring' : 'border-gray-200' ]"
/>
```

Or something like this! Not necessary to change, though.

Reply:
I like it!